miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSampleController.java

```diff
@@ -157,6 +158,9 @@
   @Autowired
   private SampleValidRelationshipService sampleValidRelationshipService;
 
+  @Autowired
+  private ProjectStore projectStore;
```

Comment:
I think the controller is only supposed to access the service layer/RequestManager.